packages/base/src/panelview/components/layers.tsx

```diff
@@ -434,7 +434,7 @@ function LayerComponent(props: ILayerProps): JSX.Element {
       >
         <LabIcon.resolveReact
           icon={layer.visible ? visibilityIcon : nonVisibilityIcon}
-          className={LAYER_ICON_CLASS}
+          className={LAYER_ICON_CLASS + (layer.visible ? '' : ' jp-mod-hidden')}
```

Comment:
```suggestion
         className={`${LAYER_ICON_CLASS} ${layer.visible ? '' : ' jp-mod-hidden'}`}
```
Sorry to be nitpicky, but we're already using literals for conditional classnames (eg: [here](https://github.com/geojupyter/jupytergis/blob/bdab589a34e5b1293f7ec421eabc28e8f96be1d3/packages/base/src/panelview/components/layers.tsx#L293) and [here](https://github.com/geojupyter/jupytergis/blob/bdab589a34e5b1293f7ec421eabc28e8f96be1d3/packages/base/src/panelview/components/layers.tsx#L408)) so I think we should be consistent about it.

Reply:
I'm totally cool with nitpicking for consistency :) However, we actually are using both approaches fairly equally in this file:

https://github.com/geojupyter/jupytergis/blob/bdab589a34e5b1293f7ec421eabc28e8f96be1d3/packages/base/src/panelview/components/layers.tsx#L300

https://github.com/geojupyter/jupytergis/blob/bdab589a34e5b1293f7ec421eabc28e8f96be1d3/packages/base/src/panelview/components/layers.tsx#L437

I'll clean up the whole file to use template literals for generated `className`s.